app/javascript/App/components/CalendarPage/CalendarList.js

```diff
@@ -27,7 +27,7 @@ class CalendarList extends Component {
 
```

Comment:
Does this file also need `import withDevice` and a corresponding HOC wrapper in the `export default`?

Reply:
Yes it does, missed that change